### DISCUSSION TITLE
openDelay on Tooltip

### DISCUSSION THREAD

**Discussion by cx0onl** [mantinedev/mantine#444] [Feedback]
It would be cool if the tooltip component also has an openDelay.

That would make the UI feel a bit "smoother" if not every tooltip would pop up immediately on hover.

Regards Joachim

**Comment by rtivital** [comment]
Thanks for proposal, we will be adding this feature in one of 3.x releases

**Reply by UltraInstinct05** [reply]
@rtivital Is this already added (I don't see anything of that sort in `next-minor`, but just making sure) ? If not, I have a local branch with necessary changes that adds an `openDelay` prop.

**Reply by rtivital** [reply]
No, I've decided that it will be a breaking change, and thus it will be a part of 4.0 release.

**Reply by UltraInstinct05** [reply]
Just curious to know how is it a breaking change if it's set to a default of `0` ? I have just adapted `handleClose`'s code to `handleOpen` to make it work (but perhaps I am glossing over some thing else ?)

**Reply by rtivital** [reply]
It is a breaking change because we need to rename current `delay` prop to `closeDelay`
